engine/uwsgi.ini

```diff
@@ -1,23 +1,100 @@
+; good resources for uwsgi configuration
+; https://www.bloomberg.com/company/stories/configuring-uwsgi-production-deployment/
+; https://uwsgi-docs.readthedocs.io/en/latest/ThingsToKnow.html
+
 [uwsgi]
+; fail to start if any parameter in the configuration file isn’t explicitly understood by uWSGI
 strict=true
+
+; This parameter prevents uWSGI from starting if it is unable to find or load your application module. Without this
+; option, uWSGI will ignore any syntax and import errors thrown at startup and will start an empty shell that will return
+; 500s for all requests. This is especially problematic because monitoring systems may observe that uWSGI started
+; successfully and think the application is available to service requests when, in fact, it is not.
+;
+; uWSGI continues to start without your application loaded because it thinks you may load an application dynamically
+; later. This is the default behavior because the dynamic loading of apps used to be common.
+need-app=true
+
 chdir=/etc/app
 module=engine.wsgi:application
+
+; The master uWSGI process is necessary to gracefully re-spawn and pre-fork workers, consolidate logs, and manage many
+; other features (shared memory, cron jobs, worker timeouts…). Without this feature on, uWSGI is a mere shadow of its
+; true self.
+;
+; This option should always be set to ‘on’ unless you are using the more complex “emperor” system for multi-app
+; deployments or are debugging specific behavior for which you want uWSGI to be limited.
 master=True
+
 pidfile=/tmp/project-master.pid
 http=0.0.0.0:8080
 processes=5
-harakiri=620
-max-requests=5000
```

Comment:
bumped `max-requests` down to `1000` and modified `harakiri` to `15` (setting it to 620 = 10m20s, probably not what we intended)